apps/rush-lib/src/cli/scriptActions/GlobalScriptAction.ts

```diff
@@ -104,6 +104,17 @@ export class GlobalScriptAction extends BaseScriptAction {
       customParameter.appendToArgList(customParameterValues);
     }
 
+    for (let i: number = 0; i < customParameterValues.length; i++) {
+      let customParameterValue: string = customParameterValues[i];
+      customParameterValue = customParameterValue.replace(/"/g, '\\"');
+
+      if (customParameterValue.indexOf(' ') >= 0) {
```

Comment:
Are there any other reasons we would want to wrap the provided string in quotes (special chars?)

Reply:
I'm pretty sure spaces are the only characters that matter.